src/mgr/MgrStandby.cc

```diff
@@ -52,6 +52,7 @@ MgrStandby::MgrStandby(int argc, const char **argv) :
   log_client(g_ceph_context, client_messenger.get(), &monc.monmap, LogClient::NO_FLAGS),
   clog(log_client.create_channel(CLOG_CHANNEL_CLUSTER)),
   audit_clog(log_client.create_channel(CLOG_CHANNEL_AUDIT)),
+  stats_clog(log_client.create_channel(CLOG_CHANNEL_STATS)),
```

Comment:
missing from this commit

Reply:
I don't think so. For some reason, I think github is just mangling the commit order:


145ca1a27a (gh-rw/wip-clog-log-level, wip-clog-log-level) mgr: log pg stats to 'stats' cluster log channel
00f44c792b mgr: add 'stats' cluster log channel
d801bdb446 mon: add 'stats' cluster log channel
9eeabf3cc9 common/LogEntry: drop check dating back to 0.87
b2aa442e88 Revert "mgr/DaemonServer: log pgmap usage to cluster log"

Reply:
(that, fwiw, is `git log --oneline`, bottom to top is the order)

Reply:
ah, i hate it when github does that!